assets/css/glossary.scss

```
@@ -0,0 +1,265 @@
+---
+---
+
+body {
+    background-attachment: fixed;
+    overflow: auto;
+    overflow-x: hidden;
+}
+
+/* For larger devices */
+@media screen and (min-width: 769px) {
+
+    .button-ltr,
+    .button-rtl {
+        width: 134px;
+        height: 58px;
+        padding-top: 22px;
+        padding-left: 68px;
+        font-family: 'Architects Daughter', 'Helvetica Neue', Helvetica, Arial, serif;
+        font-size: 23px;
+        line-height: 1.2;
+        color: white;
+        text-decoration: none;
+        vertical-align: top;
+        background: url({{ site.baseurl}}/static/github-button.png) 0 0 no-repeat;
```

we don't use the `site.baseurl` here, I think the preferred syntax here would be:

```suggestion
        background: url({{ "/static/github-button.png" | relative_url }}) 0 0 no-repeat;
```

---

I didn't realise these were equivalent (the two read very differently to me, but your version works).